agent/engine/docker_image_manager.go

```diff
@@ -72,6 +74,19 @@ func NewImageManager(cfg *config.Config, client DockerClient, state dockerstate.
 	}
 }
 
+// WhitelistImage add the image into whitelists that won't be cleaned up by agent
```

Comment:
Whatever terms we pick, we should be consistent across methods/interfaces/structs. e.g. `persistentImages`, `PersistImage` or `whitelistedImages`, `WhitelistImage`.